### DISCUSSION TITLE
How to increase quality of equipment data?

### DISCUSSION THREAD

**Discussion by djong1** [djong1/gp3s#213] [Ideas]
Users can submit new items to the database, causing incomplete data and doubles. We would like to increase the quality and lower the labour from both users and administrators.

Issues: 

1. Manufacturers are unresponsive or lazy to deliver every year an updated equipment list, based on a standardized excel that can be automatically imported.
2. Sponsored riders are not willing to update the equipment of their sponsors once a year.
3. Entering all equipment by Admin is too much work for gp3s to handle.
4. Boards submission page has currently an edit feature enabled, to edit listings and audit changes. This feature is hardly used by gp3s riders, so leaving it to the community will also not work.
5. Automated rules to discover doubles or mismatches are in place but either generate false positives or miss doubles. Manual intervention is needed. 
6 UI was using a free format search using a search engine and category filters but this was also not working in the desired way because when phrasing the search question not right would not show the desired result. Needed some fine tuning, but given the cost we also decided to decommission this feature.

<img width="663" alt="image" src="https://github.com/djong1/gp3s/assets/25177991/e10403b7-71ca-4bd2-9dec-deb723c64697">

Way Forward.
I would love to get input how to achieve a better quality for the equipment data. Please submit your ideas!

**Comment by djong1** [comment]
I have deployed an update of the MY BOARDS page, when you add new equipment to the database you get now an autocomplete listing of the equipment names already in the Database. The goal is to get less variants of the same and help users to recognize the already submitted variants and choose between the available options.

<img width="686" alt="image" src="https://github.com/djong1/gp3s/assets/25177991/647870ac-2f18-4306-bf7d-7a7edf72d86c">

This is just a little step in the right direction, but I would love to get feedback. Remember only implemented for Boards for now.

**Reply by jhdebruin** [reply]
@djong1 Has this been deployed yet ?

**Comment by djong1** [comment]
Yes

Verzonden vanuit Outlook voor iOS<https://aka.ms/o0ukef>
________________________________
Van: Jan Hendrik de Bruin ***@***.***>
Verzonden: Monday, September 4, 2023 6:38:24 PM
Aan: djong1/gp3s ***@***.***>
CC: Dylan de Jong ***@***.***>; Mention ***@***.***>
Onderwerp: Re: [djong1/gp3s] How to increase quality of equipment data? (Discussion #213)


@djong1<https://github.com/djong1> Has this been deployed yet ?

**Comment by jhdebruin** [comment]
@dylan I think with this deployment the default Add gear (of existing boards) somehow has been affected, by pressing the green + sign has been affected. This seems not to be working right now (see screenshot), when selecting it no further action occurs  (see screenshot below)
<img width="902" alt="Screenshot 2023-09-04 at 18 48 22" src="https://github.com/djong1/gp3s/assets/109060128/6fbc7b2c-f2f4-4ff9-aae1-4bef707f6c32">

**Comment by djong1** [comment]
Can you test again, should be fixed in new deployment I did this morning

**Comment by jhdebruin** [comment]
@djong1  Good ! For me its working as expected again , i've asked one other rider to test it again....

**Comment by jhdebruin** [comment]
All good now
